src/renderers/dom/client/ReactMount.js

```diff
@@ -294,7 +313,9 @@ function batchedMountComponentIntoNode(
   shouldReuseMarkup,
   context
 ) {
-  var transaction = ReactUpdates.ReactReconcileTransaction.getPooled();
+  var transaction = ReactUpdates.ReactReconcileTransaction.getPooled(
+    /* forceHTML */ shouldReuseMarkup
```

Comment:
I think in the cases where we've labeled args like this, we've done it after, not before (though I could be wrong, not sure if we actually have any in React).